### PR TITLE
Update to outdated files in the dev-1.17-ko.1

### PR DESCRIPTION
**Pull request by ysyukr**
### 28 out of 28 files to be modified
- [x] content/ko/_index.html
- [x] content/ko/docs/concepts/architecture/cloud-controller.md
- [x] content/ko/docs/concepts/architecture/nodes.md
- [x] content/ko/docs/concepts/overview/what-is-kubernetes.md
- [x] content/ko/docs/concepts/services-networking/endpoint-slices.md
- [x] content/ko/docs/concepts/workloads/controllers/daemonset.md
- [x] content/ko/docs/concepts/workloads/controllers/statefulset.md
- [x] content/ko/docs/concepts/workloads/controllers/ttlafterfinished.md
- [x] content/ko/docs/concepts/workloads/pods/pod-topology-spread-constraints.md
- [x] content/ko/docs/reference/_index.md
- [x] content/ko/docs/reference/glossary/cloud-provider.md
- [x] content/ko/docs/reference/glossary/etcd.md
- [x] content/ko/docs/reference/kubectl/cheatsheet.md
- [x] content/ko/docs/reference/using-api/client-libraries.md
- [x] content/ko/docs/setup/_index.md
- [x] content/ko/docs/setup/best-practices/certificates.md
- [x] content/ko/docs/setup/learning-environment/minikube.md
- [x] content/ko/docs/setup/production-environment/container-runtimes.md
- [x] content/ko/docs/setup/production-environment/tools/kops.md
- [x] content/ko/docs/setup/production-environment/windows/user-guide-windows-containers.md
- [x] content/ko/docs/tasks/access-application-cluster/configure-access-multiple-clusters.md
- [x] content/ko/docs/tasks/access-application-cluster/configure-dns-cluster.md
- [x] content/ko/docs/tasks/administer-cluster/cluster-management.md
- [x] content/ko/docs/tasks/manage-kubernetes-objects/kustomization.md
- [x] content/ko/docs/tasks/run-application/horizontal-pod-autoscale-walkthrough.md
- [x] content/ko/docs/tasks/run-application/horizontal-pod-autoscale.md
- [x] content/ko/docs/tasks/tools/install-minikube.md
- [x] content/ko/docs/tutorials/configuration/configure-redis-using-configmap.md

from #18049 
/language ko

### CODE REVIEW

File: content/ko/docs/setup/production-environment/windows/user-guide-windows-containers.md [hunk 1]
```diff
@@ -120,25 +120,123 @@ spec:
...
+    kubernetes.io/arch: 'amd64'
+    node.kubernetes.io/windows-build: '10.0.17763'
+  tolerations:
+  - effect: NoSchedule
+    key: os
+    operator: Equal
+    value: "windows"
+```
+
+2. 클러스터 관리자로 `kubectl create -f runtimeClasses.yml` 를 실행해서 사용한다
```
Comment (seokho-son) [inline content/ko/docs/setup/production-environment/windows/user-guide-windows-containers.md]:
```suggestion
2. 클러스터 관리자로 `kubectl create -f runtimeClasses.yml` 를 실행해서 사용한다.
```

File: content/ko/docs/setup/production-environment/windows/user-guide-windows-containers.md [hunk 2]
```diff
@@ -120,25 +120,123 @@ spec:
...
+3. 파드 사양에 적합한 `runtimeClassName: windows-2019` 를 추가한다
```
Comment (seokho-son) [inline content/ko/docs/setup/production-environment/windows/user-guide-windows-containers.md]:
```suggestion
3. 파드 사양에 적합한 `runtimeClassName: windows-2019` 를 추가한다.
```

File: content/ko/docs/tasks/tools/install-minikube.md [hunk 3]
```diff
@@ -186,6 +194,41 @@ Minikube 설치를 마친 후, 현재 CLI 세션을 닫고 재시작한다. Mini
...
+minikube start --vm-driver=<driver_name>
+```
+
+`minikube start` 가 완료되면, 아래 명령을 실행해서 클러스터의 상태를 확인한다.
+
+```shell
+minikube status
+```
+
+만약 클러스터가 실행중이면, `minikube status` 의 출력은 다음과 유사해야한다.
```
Comment (seokho-son) [inline content/ko/docs/tasks/tools/install-minikube.md]:
```suggestion
만약 클러스터가 실행 중이면, `minikube status` 의 출력은 다음과 유사해야 한다.
```

**Comment by ysyukr** [comment]
@seokho-son 리뷰 감사합니다. 제안해주신 부분은 확인해서 반영했습니다.

**Comment by seokho-son** [comment]
Thank you!
/lgtm
/approve

cc @gochist 
This PR has been approved !

**Pull request merged**